### ISSUE TITLE
ZoomConnect missing securityDefs

### ISSUE THREAD

**Issue by bobby-brennan** [APIs-guru/openapi-directory#137]
Details here: https://zoomconnect.com/api

It should have:
```json
{
  "securityDefinitions": {
    "email": {
      "type": "apiKey",
      "in": "header",
      "name": "email"
    },
    "token": {
      "type": "apiKey",
      "in": "header",
      "name": "token"
    }
  }
}
```

**Comment by IvanGoncharov** [comment]
@bobby-brennan Fixed in 89742730e788ef3cae70675ff5bbae22abb0d182

**Comment by bobby-brennan** [comment]
Thanks!

Bobby Brennan
bobby.brennan@gmail.com
